### DISCUSSION TITLE
Getting Gibberish File Listings in Transfer Utility After 13 Files

### DISCUSSION THREAD

**Discussion by tychoc** [BlueSCSI/BlueSCSI-v2#96] [Help] [answered]
The Blue SCSI SD Transfer Utility will not list more than 13 files for me. All additional file names show as gibberish or blank. Attempting to transfer these files results in various errors or system lock-up. 

Tested with an 8GB microSD freshly zeroed using the SD Card Formatter utility, SD Transfer Utility 1.0.0, System 7.1u3 and 7.5, with the 23-10-13 firmware, on two different machines. 

The log file reports the correct numerical count of files, but also lists all files after 13 as gibberish or blank. 

![IMG_0876](https://github.com/BlueSCSI/BlueSCSI-v2/assets/18652454/f999a0b4-967a-4e29-9e82-66fb404b30ea)
[debuglog_.txt](https://github.com/BlueSCSI/BlueSCSI-v2/files/13328742/debuglog_.txt)

**Comment by erichelgeson** [comment]
This usually happens when a file name has a character not in mac-roman or some other weirdness. Could you share the list of files in the shared folder? A screen shot from a modern computer or even zipping up the shared dir would be helpful in debugging.

**Reply by erichelgeson** [reply, marked as answer]
Actually never mind! I found one buffer was set to 512 where it should be set to 4096, hence truncating the response to ~13 files. I should be able to build a new 1.0.1 in the next day or two to resolve this. Thanks for the report!

**Reply by tychoc** [reply]
Beautiful! Happy to help, and thanks for all your work!

**Reply by erichelgeson** [reply]
https://bluescsi.com/toolbox - grab 1.0.1 from here
